src/main/java/Ejercicio3/Ejercicio2.java

```diff
@@ -0,0 +1,19 @@
+package Ejercicio3;
+
+import java.util.Arrays;
+
+public class Ejercicio2 {
+    public static void main(String[] args) {
+        final var arreglo = Ejercicio1.crearArregloAleatorio(500);
+        System.out.printf("El arreglo multiplicado por 2 es: %s%n", Arrays.toString(Ejercicio2.multiplica2(arreglo)));
+
+    }
+
+    private static int[] multiplica2(int[] arreglo) {
+        var nuevoArreglo = new int[500];
```

Comment:
acá estás asumiendo que siempre es 500, tiene que ser el mismo tamaño del arreglo por parámetro

Reply:
por eso sugiero declarar n
final var n = arrreglo.lenth;

final var nuevoArreglo = new int[n];

y abajo en el for lo usas en vez de arreglo.length